CMakeLists.txt

```diff
@@ -1076,6 +1087,30 @@ if (ENABLE_CGAL)
 endif()
 list(APPEND Sources src/openscad.cc ${CORE_SOURCES} ${OFFSCREEN_SOURCES})
 
+set_source_files_properties(
+  ${OFFSCREEN_SOURCES}
+  ${PLATFORM_SOURCES}
+  ${FLEX_openscad_lexer_OUTPUTS}
+  ${BISON_openscad_parser_OUTPUTS}
+  ${FLEX_comment_lexer_OUTPUTS}
+  ${BISON_comment_parser_OUTPUTS}
+  ${MANIFOLD_SOURCES}
+  src/glview/OffscreenContextFactory.cc
+  src/glview/RenderSettings.cc
+  src/glview/Camera.cc
+  src/glview/ColorMap.cc
+  src/glview/preview/CSGTreeNormalizer.cc
+  src/gui/input/SpaceNavInputDriver.cc
+  src/gui/QGLView.cc
+  src/gui/QGLView2.cc
+  # things that requires _USE_MATH_DEFINES
+  src/core/BuiltinContext.cc
+  src/geometry/roof_vd.cc
+  src/io/DxfData.cc
+  src/utils/calc.cc
+  src/utils/degree_trig.cc
+  PROPERTIES SKIP_UNITY_BUILD_INCLUSION ON)
```

Comment:
List list of exclusions is hard to read and hard to maintain. We'd probably want some reasoning per file for why it needs to be excluded, to make it possible to add to or remove from this list in the future.